### DISCUSSION TITLE
Hold mouse button to attack - QoL improvement.

### DISCUSSION THREAD

**Discussion by Darmorque** [diasurgical/DevilutionX#3113] [Ideas]
Would that be possible to implement such thing? I used to play GOG version of Diablo/Hellfire with dClicker enabled and I have to admit, that it made the gameplay way more in line with Diablo 2/3. Holding either mouse button to attack or cast spells instead of clicling makes it really enjoyable experience. It would be especially helpful considering game speeed setting in DevilutionX, when furious mouse clicking at higher speeds is a must. Poor mouse. ;) What do you think about it? Isn't that worth considering?

**Comment by qndel** [comment]
It's already there, will be available in 1.3.0 or you can grab one of the "test" builds from github

**Reply by Darmorque** [reply]
Tried one of the latest test builds. Works like a charm. Awesome. Thanks for your work!
